debian/bullseye/foreman/control

```diff
@@ -227,3 +216,16 @@ Description: package providing dynflow sidekiq services for Foreman
  a flexible systems management web application.
  .
  The foreman package should be installed along with this package.
+
+Package: foreman-google
```

Comment:
No, the package still needs to be called foreman-gce, as that's what the users have installed right now. And it needs a "Depends" on ruby-foreman-google, which is the plugin package.

The breaks etc only need to be in the plugin package.

Reply:
Reverted back